lib/ohai/plugins/windows/memory.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Author:: Sean Escriva (<sean.escriva@gmail.com>)
+# Copyright:: Copyright (c) 2014 Sean Escriva
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:Memory) do
+  provides "memory"
+
```

Comment:
I recommend adding a `depends "os"`. Likely, the os plugin will always be run first but it never hurts to be safe.